examples/istioctl-analyze/README.md

```diff
@@ -0,0 +1,37 @@
+# Istioctl Analyze
+
+The `istioctl-analyze` config function detects potential issues with your Istio
+configuration and output errors as results. This example invokes the
+istioctl-analyze function using declarative configuration.
+
+## Function invocation
+
+The function is invoked using the function configuration in
```

Comment:
General structure to follow for example blueprints:

1. Command to run
2. Expected results
3. Description of anything non-obvious or unique to this function.

Reply:
Having an introduction before giving commands is more user-friendly. I am keeping an intro with Step 3 first, then Step 1 & Step 2.